packages/react-reconciler/src/__tests__/__snapshots__/ReactIncrementalPerf-test.internal.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ReactDebugFiberPerf captures all lifecycles 1`] = `
-"⚛ (Waiting for async callback...)
+"⚛ (Waiting for async callback... timeout is 5230)
```

Comment:
Maybe we can make it clearer what this number means?

e.g. `(Waiting for async callback... will force flush in 5320ms)`

Reply:
I like that! Will fix.